### PR TITLE
DragonFruit argument support

### PR DESCRIPTION
**Pull request by jonsequitur**
This addresses #297.

### CODE REVIEW

File: src/System.CommandLine.DragonFruit/CommandLine.cs [hunk 1]
```diff
@@ -116,6 +116,18 @@ public static class CommandLine
                 command.AddOption(option);
             }
 
+            if (method.GetParameters()
```
Comment (Keboo) [inline src/System.CommandLine.DragonFruit/CommandLine.cs]: This ends up with the parameter being included as both an argument and an option. Is that what was intended? I think at a minimum we should have another test around this behavior (either including these parameters as options or not).

File: src/System.CommandLine.DragonFruit/CommandLine.cs [hunk 2]
```diff
@@ -116,6 +116,18 @@ public static class CommandLine
...
+                      .FirstOrDefault(p => p.Name == "arguments" || 
+                                           p.Name == "args" ||
+                                           p.Name == "argument") is ParameterInfo argsParam)
```
Comment (Keboo) [inline src/System.CommandLine.DragonFruit/CommandLine.cs]: Can we change the method signature to take in an optional collection of names to match on for arguments, and fall back to using this set if that collection is not specified. That would also give (albeit non-obvious) method for disabling/configuring this behavior.

File: src/System.CommandLine.DragonFruit/CommandLine.cs (hunk 1 above)
Comment (jonsequitur) [reply src/System.CommandLine.DragonFruit/CommandLine.cs]: Ah, good point. I'll fix that.

File: src/System.CommandLine.DragonFruit/CommandLine.cs (hunk 2 above)
Comment (jonsequitur) [reply src/System.CommandLine.DragonFruit/CommandLine.cs]: This functionality is only available in DragonFruit which is so far not configurable, so I'd prefer to wait on making this more flexible until we decide whether that's something to pursue.

**Pull request merged**